interface/client/templates/views/tab.js

```diff
@@ -50,14 +49,24 @@ Template['views_tab'].onRendered(function(){
         // Meteor.clearTimeout(timeoutId);
         TemplateVar.set(template, 'loading', false);
 
+        var titleFull = webview.getTitle(),
+            title;
+
+        if(titleFull && titleFull.length > 40) {
```

Comment:
Why are you cropping it on javascript? This could be accomplished with css, using `overflow:hidden; text-overflow: ellipsis; white-space:nowrap;`. Word width is not the same as character count.